### PR TITLE
"added the button fix as per ui"

### PR DESCRIPTION
**Pull request by samcaspus**
Fixes [#801](https://github.com/creativecommons/cccatalog-frontend/issues/801)

Made a minor change of colour as requested for the mockup
It has addition of an image for the same purpose (Back logo)

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by kgodey** [comment]
We don't add images directly here, we need to add the icon to Vocabulary first. @dhruvkb can help with that.

**Comment by samcaspus** [comment]
Actually new to this so how am I suppose to proceed further with this?

File: src/components/PhotoDetails.vue [hunk 1]
```diff
@@ -4,7 +4,13 @@
       <a class="photo_breadcrumb"
           :href="breadCrumbURL"
           @click.prevent="onGoBackToSearchResults"
-          v-if="shouldShowBreadcrumb">&#171; Back to search results</a>
+          v-if="shouldShowBreadcrumb">
+          <strong>
+            <font color="grey">
+              &olt; Back to search results
```
Comment (dhruvkb) [inline src/components/PhotoDetails.vue]: Impressive use of `&olt;`! Learned something today.

File: src/components/PhotoDetails.vue [hunk 2]
```diff
@@ -4,7 +4,13 @@
       <a class="photo_breadcrumb"
           :href="breadCrumbURL"
           @click.prevent="onGoBackToSearchResults"
-          v-if="shouldShowBreadcrumb">&#171; Back to search results</a>
+          v-if="shouldShowBreadcrumb">
+          <strong>
+            <font color="grey">
```
Comment (dhruvkb) [inline src/components/PhotoDetails.vue]: The `font` tag has been deprecated.

Comment (dhruvkb) [reply src/components/PhotoDetails.vue]: Always use CSS for styling, in this case the `color` property.

**Comment by dhruvkb** [comment]
@brenoferreira I think we can close this.

**Pull request closed without merging**